### PR TITLE
App API update for new properties

### PR DESCRIPTION
**Pull request by sureshja**
@lleonard-msft, @dkershaw10 - Please review. This update contains description for new properties being added for the next beta update. Also minor fixes for parental control settings and preauthorized applications.

### CODE REVIEW

File: api-reference/beta/resources/api.md [hunk 1]
```diff
@@ -8,8 +8,11 @@ Specifies settings for a Web API application.
 
 | Property | Type | Description |
 |:---------------|:--------|:----------|
-|requestedAccessTokenVersion|Int32| Specifies the accepted access token version for the current API resource. Possible values are 1 or 2.  |
+|acceptMappedClaims|Boolean| When true, allows an application to use claims mapping without specifying a custom signing key. |
```
Comment (dkershaw10) [inline api-reference/beta/resources/api.md]: What's the default behavior?  What happens if this is not set?  
Is there more information you can link to or help with around specifying a **custom signing key** - where and how do I do that?

File: api-reference/beta/resources/api.md [hunk 2]
```diff
@@ -8,8 +8,11 @@ Specifies settings for a Web API application.
...
+|knownClientApplications|String collection| Used for bundling consent if you have a solution that contains two parts: a client app and a custom web API app. If you enter the appID of the client app into this value, the user will only have to consent once to the client app. Azure AD will know that consenting to the client means implicitly consenting to the web API and will automatically provision service principals for both the client and web API at the same time. Both the client and the web API app must be registered in the same tenant.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/api.md]: I wonder if we can tighten this up and shorten it
_If you enter the appID of the client app into this value, the user will only have to consent once to the client app. Azure AD will know that consenting to the client means implicitly consenting to the web API and will automatically provision service principals for both the client and web API at the same time._

How about:
For a web API application definition, setting this property to the appID of an associated client app, means that when a user consents to the client app they will also be consenting to the web API app at the same time. Permissions requested by both the client and the web API apps will be displayed in the consent dialog.

If you want to add some stuff about service principals being provisioning and that the client and web API apps must be in the same tenant, that fine to keep.

File: api-reference/beta/resources/api.md [hunk 3]
```diff
@@ -8,8 +8,11 @@ Specifies settings for a Web API application.
...
 |oauth2PermissionScopes|[permissionScope](permissionscope.md) collection| The collection of OAuth 2.0 permission scopes that the web API (resource) application exposes to client applications. These permission scopes may be granted to client applications during consent. |
+|preAuthorizedApplications|[preAuthorizedApplication](preAuthorizedApplication.md)| Specifies the list of applications and requested permissions to be pre-authorized for implicit consent. Authorizing a client application indicates that this API trusts the application and users will not be asked to consent when the client calls this API. Only the permissions listed in preAuthorizedApplications do not require user consent. Any additional requested permissions not listed in preAuthorizedApplications require user consent. Requires an admin to have provided consent to the application. |
```
Comment (dkershaw10) [inline api-reference/beta/resources/api.md]: Good explanation, until the very final sentence.  I don't understand what _Requires an admin to have provided consent to the application._ means in this context.  Please fix

File: api-reference/beta/resources/api.md [hunk 4]
```diff
@@ -24,8 +27,11 @@ Here is a JSON representation of the resource.
 
 ```json
 {
-  "requestedAccessTokenVersion": 1,
-  "oauth2PermissionScopes": [{"@odata.type": "microsoft.graph.permissionScope"}]
+  "acceptMappedClaims": true,
+  "knownClientApplications": ["String (identifier)"],
+  "oauth2PermissionScopes": [{"@odata.type": "microsoft.graph.permissionScope"}],
+  "preAuthorizedApplications": [{"@odata.type": "microsoft.graph.PreAuthorizedApplication"}],
+  "requestedAccessTokenVersion": 1
```
Comment (dkershaw10) [inline api-reference/beta/resources/api.md]: Should `requestedAccessTokenVersion` be removed?  It looks like it was removed in the property table?

**Comment by dkershaw10** [comment]
@sureshja Added comments. Please make some updates to the PR.

**Comment by Lauragra** [comment]
@davidmu1 , can you please follow up on this PR? It has open comments that need to be resolved. We'll need to open a new PR in order to merge in these changes. Thanks.

**Comment by davidmu1** [comment]
Suresh is updating application articles for GA. Work will be done in a new PR.

**Pull request closed without merging**